### PR TITLE
Crashed elyran tanker

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image
image

.

Features:
- Away site, no ghost roles.
- Can't be made into a functioning ship.
- Partially pressurized. The south broken up interior part has no air and is exposed to space. While the north parts, docking arm, bridge, crew area, are pressurized.
- Four exterior navigation landmarks, and two docking port navmarks. The docking port airlocks are not functional. It's a crashed, ruined ship, things are not going to work as they should.
- Has areas mapped in.
- Has power cables and APCs mapped in. APCs are all drained/empty, and the wires are all broken up.
- Has firedoors and fire alarms.
- No atmos pipes and vents/scrubbers mapped in, no functioning atmos system. Didn't think it was really worth it. There are some portable scrubbers and air pumps though, so like, maybe they use those instead.
- Weapons: Plasma pistol, double-barrel shotgun. That's it, if I did not forget anything else. It's not proper military ship.
- Valuable loot: Pretty much just phoron. It has a big pressurized phoron tank, but it requires fixing up the pipes, and providing power, either by bringing a portable generator or fixing the solars. Ideally an engineer would get involved in fixing this.

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!wip

**Comment by WickedCybs** [comment]
Files are too big to view on the website but if it was not done yet, I'd suggest mapping in multiple areas for the site. Helps with future changes among other things.

**Comment by DreamySkrell** [comment]
!wip

**Comment by DreamySkrell** [comment]
!review

File: code/game/machinery/floodlight.dm [hunk 1]
```diff
@@ -134,3 +134,10 @@
 				to_chat(user, SPAN_NOTICE("You insert the power cell."))
 		return TRUE
 	update_icon()
+
+/obj/machinery/floodlight/randomcharge
```
Comment (SleepyGemmy) [inline code/game/machinery/floodlight.dm:138]: wrong path name?

File: code/game/machinery/floodlight.dm [hunk 2]
```diff
@@ -134,3 +134,10 @@
...
+
+/obj/machinery/floodlight/Initialize()
```
Comment (SleepyGemmy) [inline code/game/machinery/floodlight.dm:140]: wrong path name?

Comment (DreamySkrell) [reply code/game/machinery/floodlight.dm]: Fixed.

File: code/game/machinery/floodlight.dm [hunk 3]
```diff
@@ -134,3 +134,10 @@
 				to_chat(user, SPAN_NOTICE("You insert the power cell."))
 		return TRUE
 	update_icon()
+
+/obj/machinery/floodlight/randomcharge
+
```
Comment (SleepyGemmy) [inline code/game/machinery/floodlight.dm:139]:
```suggestion
	// Intentionally left empty as it's the same as the parent, but the cell is randomized.
```

File: code/game/machinery/floodlight.dm [hunk 4]
```diff
@@ -134,3 +134,10 @@
 				to_chat(user, SPAN_NOTICE("You insert the power cell."))
 		return TRUE
 	update_icon()
+
+/obj/machinery/floodlight/randomcharge
+	// Intentionally left empty as it's the same as the parent, but the cell is randomized.
```
Comment (SleepyGemmy) [inline code/game/machinery/floodlight.dm:139]:
```suggestion
	// Intentionally left empty as it's the same as the parent, but the cell is randomized.

```

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**